crates/api_common/src/comment.rs

```diff
@@ -131,6 +131,7 @@ pub struct GetComments {
   pub post_id: Option<PostId>,
   pub parent_id: Option<CommentId>,
   pub saved_only: Option<bool>,
+  pub liked_only: Option<i16>,
```

Comment:
The name should also be changed to `voted_only`. Alternatively, use two boolean settings `liked_only` and `disliked_only`.

Reply:
Yeah, I've noticed this and created issue #3524 to handle it from start to finish. Currently I just wanted to stay consistent with the type used for indicating the like type. Do you think perhaps working on #3524 first is the way to go? Using an enum would lead to better clarity.

Reply:
I've made a comment in #3524 , that would probably break summing, so I wouldn't wait on that.

Reply:
Also, it should be a `Option<bool>` , not a number or an enum. And if you should probably add two fields: `liked_only` and `disliked_only`